lesson29/src/resetingemail.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="ja">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <link rel="stylesheet" href="./css/reset.css" type="text/css">
+        <link rel="stylesheet" href="./css/style.css" type="text/css">
+        <script type="module" src="./js/changeemail.js"></script>
+        <script src="./js/contents/main.js" defer></script>
+        <title>メールアドレス変更ページ</title>
+    </head>
+    <body>
+        <script>
+            if (!localStorage.getItem("token")) window.location.href = "./login.html";
+        </script>
+
+        <header class="header">
+            <div class="header__inner">
+                <h1 class="title title--top"><a href="./" class="header__link">haru news</a></h1>
+                <button class="header__button header__link" id="js-usermenu-button" type="button"><img src="./img/img-user.png" alt="ユーザー管理画面" width="60" height="60" decoding="async"></button>
+            </div>
+            <div class="usermenu" id="js-usermenu" role="dialog">
+                <button class="usermenu__icon" id="js-close-icon" aria-label="Close">&times;</button>
+                <span class="usermenu__img"><img src="./img/img-user.png" alt="ユーザー画像"></span>
+                <p class="usermenu__name" id="js-username"></p>
+                <p class="usermenu__email" id="js-email"></p>
+                <ul class="usermenu__list">
+                    <li class="usermenu__item"><a href="./resetingemail.html" class="usermenu__link">メールアドレス変更</a></li>
+                </ul>
+                <button class="usermenu__button" id="js-logout-button" type="button">Logout</button>
+            </div>
+        </header>
+        <div class="setting">
+            <div class="setting__inner">
+                <div class="setting__menu">
+                    <span class="setting__img"><img src="./img/img-user.png" alt="ユーザー画像"></span>
+                    <ul class="setting__menu-list">
+                        <li class="setting__menu-item is-active">メールアドレス変更</li>
+                        <li class="setting__menu-item"><a href="#" class="usermenu__link">パスワード変更</a></li>
+                    </ul>
+                </div>
+                <div class="scallop-down"></div>
+                <div class="setting__body">
+                    <form class="form">
+                        <h2 class="title title--green">▶︎ Change email address</h2>
+                        <div class="form__item">
+                            <label for="email">新規E-mailアドレス入力<span class="form__required">必須</span></label>
+                            <input type="email" class="form__item-input js-form-email" id="email" name="email" required>
```

Comment:
nits
It may be more user-friendly to disable autocomplete for new email address entry fields.
However, this does not seem to work with Google Chrome, so don't worry about it too much.
<img width="500" alt="スクリーンショット 2023-01-14 11 04 48" src="https://user-images.githubusercontent.com/99260576/212445979-7ea0ddef-3dba-43e5-be52-537737d96d70.png">

```suggestion
                            <input type="email" class="form__item-input js-form-email" id="email" name="email" autocomplete="off" required>
```

Reply:
Thank you for your review!
As you taught us, we felt it would be easier for users if autocomplete was disabled.
Sorry for the delay after Approve, but we have made the correction.🙇‍♀️
[0f12316](https://github.com/haru-programming/JavaScript-lessons/pull/38/commits/0f12316d5cbbfbbe5a72c2fd2304b9489a38a2a6)